CoreScriptsRoot/Modules/VR/Recenter.lua

```diff
@@ -72,6 +78,12 @@ function RecenterModule:SetVisible(visible)
 			isCountingDown = true
 			countdownPanel:SetVisible(true)
 
+			ContextActionService:BindCoreAction(cancelShortcutName, function(actionName, inputState, inputObj)
+				if inputState == Enum.UserInputState.Begin then
+					cancelCountdown()
+				end
+			end, false, Enum.KeyCode.ButtonB)
+
```

Comment:
Ah sorry, github for some reason folded the code for 1 line, which, happened to be the one with `spawn(function()` on it